docs: updating with new RC work

moxiegirl opened the pull request:
Adjusting deploying after new RC pushed
Fixing typos and adding tables
Updating with testing material
Tweak after read through
Fixes issue #363 too!

Signed-off-by: Mary Anthony <mary@docker.com>

moxiegirl commented:
ping @dmcgowan @bfirsh for review commentary  
I'd like to get this reviewed and out today --- we need to do that for it to be ready for release tomorrow.

stevvooe commented on docs/configuration.md:
```diff
@@ -142,15 +121,19 @@ N.B. In some instances a configuration option may be marked **optional** but con
 
-N.B. The version of the registry software may be found at [/version/version.go](https://github.com/docker/distribution/blob/master/version/version.go)
+>**Note**: You can find the registry software version at
+[/version/version.go](https://github.com/docker/distribution/blob/master/version
```
This file doesn't actually provide the version until after the linker runs. It only contains a reasonable guess.

moxiegirl replied:
I'm gonna remove that note it was kinda iffy anyway.

stevvooe commented on docs/configuration.md:
```diff
@@ -358,25 +744,112 @@ http:
 ### tls
 
-The tls option within http is **optional** and allows you to configure SSL for the server. If you already have a server such as Nginx or Apache running on the same host as the registry, you may prefer to configure SSL termination there and proxy connections to the registry server.
+The `tls` struct within `http` is **optional**. Use this to configure SSL
```
Not sure if this can fixed now: SSL and TLS seemed to be used interchangeably. We should only use TLS.

stevvooe commented on docs/configuration.md:
```diff
@@ -358,25 +744,112 @@ http:
 ### debug
 
-The debug option is **optional** and allows you to configure a debug server that can be helpful in diagnosing problems. It is of most use to contributers to the distribution repository and should generally be disabled in production deployments.
+The `debug` option is **optional** . Use it to configure a debug server that can
```
The debug server really exposes internal runtime state via `pprof` and `expvar`. Not sure how much detail we should have about that here. Maybe just take note/

moxiegirl replied:
I think I'll leave the details out of this.  We can circle back if we get a lot of people asking about the mechanism.

stevvooe commented on docs/deploying.md:
```diff
@@ -329,138 +328,238 @@ your base image.
+
+2. Get a registry 1.0 image.
+
+		$ docker pull registry:latest 
```
This tag should be 0.9.1 or similar.

stevvooe commented:
@moxiegirl This is looking fine. I found a few items that we may need to fix. Please take note and we'll get this merged.

moxiegirl commented:
@stevvooe You rock! Thanks for the quick review.  If you spot something later, we can pick it up in a later release.

RichardScothern commented on contrib/compose/README.md:
```diff
@@ -1,23 +1,147 @@
+		* SSL certificate problem: self signed certificate
+		* Closing connection 0
+		curl: (60) SSL certificate problem: self signed certificate
+		More details here: http://curl.haxx.se/docs/sslcerts.html
```
Make this a clickable link?

stevvooe replied:
@RichardScothern Markdown should render this.

RichardScothern replied:
I thought it required ()[] notation.  I couldn't see it with 'View File' in github, although it is in an indented grey block

moxiegirl replied:
It is part of curl's output --- it isn't intended for a reference on the page

RichardScothern commented on contrib/compose/README.md:
```diff
@@ -1,23 +1,147 @@
+			ports:
+				- "5000"
+
+ 		This configuration builds a new `nginx` image as specified by the
```
This indentation puts the paragraph in the configuration grey box.  I'm not sure if it's intentional, but you lost the `` italic effect.

RichardScothern commented on docs/configuration.md:
```diff
@@ -134,7 +110,10 @@ redis:
-N.B. In some instances a configuration option may be marked **optional** but contain child options marked as **required**. This indicates that a parent may be omitted with all its children, however, if the parent is included, the children marked **required** must be included.
+In some instances a configuration option is **optional** but it contains child
+options marked as **required**. This indicates that you can omit the parent with
+all its children. However, if the parent is included, you must alos include all
```
alos -> also

RichardScothern commented on docs/configuration.md:
```diff
@@ -328,18 +640,88 @@ reporting:
+      New Relic application name.
+    </td>
+  </tr> 
+</table>
```
There is a third option available to newrelic:
verbose bool : enable newrelic debugging output on stdout

see https://github.com/docker/distribution/commit/c359a6d1726c2547ceb5914a3a3fb3c5acaea46f

RichardScothern commented on docs/configuration.md:
```diff
@@ -393,19 +862,119 @@ notifications:
+  </tr>
+  <tr>
+    <td>
+      <code>url/code>
```
Missing '<' before /code

moxiegirl replied:
you got the eagle eye!

RichardScothern commented:
A couple of small nits, but otherwise it's looking good!

moxiegirl commented:
@RichardScothern thanks for the eagle eye! @stevvooe I fixed that link in the readme and added it to this.  Also, tweaked the image links to fix in docs.docker.com

stevvooe commented:
@moxiegirl Could you fix the indentation issue identified by @RichardScothern? This looks good otherwise.

moxiegirl commented:
ping @stevvooe sorry I though tI had that last night. Here you go

stevvooe commented:
@moxiegirl This will need a rebase.

moxiegirl commented:
ping @stevvooe done!
